features/enlinkd/pom.xml

```diff
@@ -17,5 +17,7 @@
     <module>adapters</module>
     <module>daemon</module>
     <module>tests</module>
+    <module>org.opennms.features.enlinkd.shell</module>
```

Comment:
These new modules use fully qualified names whereas the existing modules use short ones.

Reply:
Good point. I fixed the feature naming, but oversaw this. 
Probably also want to rename the `generator` module as it is also a fully qualified name.

Reply:
fixed.